en/Licenses & Payment/Commercial license.md

```diff
@@ -51,17 +51,20 @@ For bulk discounts, contact support@obsidian.md.
 > [!FAQ]- Q9: I'm a freelancer/writer/blogger, do I need a commercial license?
 > Unless you hire at least one additional person, you do not need a commercial license.
 
-> [!FAQ]- Q10: I'm a teacher/professor and I use Obsidian for my teaching/lectures, do I need a commercial license?
+> [!FAQ]- Q10: I'm a teacher/professor or I work at a school and I use Obsidian for my teaching/lectures, do I need a commercial license?
 > Obsidian usage for education-related activities within schools and other recognized educational institutions does not require a commercial license. 
 
 > [!FAQ]- Q11: I'm an academic researcher and I use Obsidian for my research work. Do I need a commercial license?
 > If your research involves producing a commercial product (e.g., if your research contributes to revenue-generating commercial activities), you need a commercial license for your usage. If your research does not contribute to a commercial product, or if your research has not yet led to commercial activities, you do not need a license.
 
-> [!FAQ]- Q12: I use Obsidian during work for things like writing down team processes and taking notes for team meetings, do I need a commercial license?
+> [!FAQ]- Q12: I work for a Government or an arms-length governmental agency. Do I need a commercial license?
```

Comment:
```suggestion
> [!FAQ]- Q12: I work for a government or an arms-length governmental agency. Do I need a commercial license?
```

I'm not aware of the term "arms-length governmental agency". Is this an idiom, or an actual term that government users would be familiar with?

Reply:
@ryanjamurphy what do you think?

Reply:
It's a term used in discussing Canadian government agencies, e.g., <https://learninglink.oup.com/protected/files/content/file/1529076398347-Chapter11_SSG.pdf>.

I am open to other ideas, though. Is "governmental agency" simpler while still covering the same cases? cc @ericaxu / @marcusolsson

Reply:
```suggestion
> [!FAQ]- Q12: I work for a government or an arm's length body or agency. Do I need a commercial license?
```

Did some research and seems like "arm's length body" is indeed a rather established term, I'm fine with using it.